### PR TITLE
sleepphasealarm

### PR DESCRIPTION
**Pull request by nxdefiant**
- Read alarms from new scheduling library
- account for higher acceleration sensor noise on Bangle.js 2

### CODE REVIEW

File: apps/sleepphasealarm/app.js [hunk 1]
```diff
@@ -38,13 +38,20 @@ function calc_ess(val) {
   }
 }
 
+// copied from alarm app
+// time in ms -> { hrs, mins }
+function decodeTime(t) {
```
Comment (nxdefiant) [inline apps/sleepphasealarm/app.js:43]: I have seen this function in alarm/app.js and
sched/sched.js. Should probably be moved to
library

**Comment by nxdefiant** [comment]
Yes, extra commit in this PR is intentional

File: apps/sleepphasealarm/app.js (hunk 1 above)
Comment (gfwilliams) [reply apps/sleepphasealarm/app.js:43]: Thanks - yes, that's a good idea. Do you want to do a PR for it?

**Comment by gfwilliams** [comment]
Thanks for switching to `sched`!

File: apps/sleepphasealarm/app.js (hunk 1 above)
Comment (nxdefiant) [reply apps/sleepphasealarm/app.js:43]: I can try that later

**Pull request merged**